kmod/core/core.c

```diff
@@ -195,6 +195,7 @@ static inline int kpatch_compare_addresses(unsigned long stack_addr,
 	return 0;
 }
 
+#if LINUX_VERSION_CODE <= KERNEL_VERSION(4, 5, 0)
```

Comment:
Would this break with a kernel 4.5.1?

Maybe it would better to say `#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 6, 0)`.  Not sure if that would exclude the 4.6-rcX releases, but we don't need to support them anyway.